iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/FileUploadTests.java

```diff
@@ -36,20 +39,20 @@
 import java.util.concurrent.*;
 import java.util.concurrent.atomic.AtomicBoolean;
 
-import static com.microsoft.azure.sdk.iot.common.helpers.CorrelationDetailsLoggingAssert.buildExceptionMessage;
-import static com.microsoft.azure.sdk.iot.common.tests.iothub.FileUploadTests.STATUS.FAILURE;
-import static com.microsoft.azure.sdk.iot.common.tests.iothub.FileUploadTests.STATUS.SUCCESS;
 import static com.microsoft.azure.sdk.iot.device.IotHubStatusCode.OK;
 import static com.microsoft.azure.sdk.iot.device.IotHubStatusCode.OK_EMPTY;
 import static junit.framework.TestCase.fail;
 import static org.junit.Assert.*;
+import static tests.integration.com.microsoft.azure.sdk.iot.helpers.CorrelationDetailsLoggingAssert.buildExceptionMessage;
+import static tests.integration.com.microsoft.azure.sdk.iot.iothub.FileUploadTests.STATUS.FAILURE;
+import static tests.integration.com.microsoft.azure.sdk.iot.iothub.FileUploadTests.STATUS.SUCCESS;
 
 /**
- * Test class containing all tests to be run on JVM and android pertaining to FileUpload. Class needs to be extended
- * in order to run these tests as that extended class handles setting connection strings and certificate generation
+ * Test class containing all tests to be run on JVM and android pertaining to FileUpload.
  */
 @FlakeyTest
 @IotHubTest
+@RunWith(Parameterized.class)
```

Comment:
This annotation used to sit at the JVMRunner level, but since that level no longer exists, this logic is here